modules/apps/client-extension/client-extension-service/src/main/java/com/liferay/client/extension/internal/events/ClientExtensionsServicePreAction.java

```diff
@@ -175,6 +193,12 @@ private String _getFaviconURL(Layout layout) {
 	}
 
 	private ThemeCSSCET _getThemeCSSCET(Layout layout) {
+		if (layout.isTypeControlPanel() &&
+			FeatureFlagManagerUtil.isEnabled("LPD-34650")) {
```

Comment:
Move the feature flag check inside the `_getControlPanelThemeCSSCET` method and return null